express-basic/server.js

```diff
@@ -11,20 +11,16 @@ const app = Express();
 
 app.use("/color/:name", function (req, res) {
   let name = req.params.name;
-  console.log("NAME", name);
+  console.log("NAME", name.toLocaleLowerCase());
 
   let code = "#cccccc";
-
-  // if('red' === name) {
-  //   code = '#ff0000'
-  // } else if('green' === name) {
-  //   code = '#00ff00'
-  // } else if('blue' === name) {
-  //   code = '#0000ff'
-  // }
-
-  //Using switch statement.
   switch (name) {
```

Comment:
this is still the original name, not the lowercase version, so `aliceblue` will not work

Reply:
I'm not sure I'm following here. Could you clarify a bit?

Reply:
![aliceblue](https://user-images.githubusercontent.com/101836990/199289386-b4d03fa9-81e0-4c9d-9042-f9859d67d986.png)

``aliceblue`` seems to work for me.

Reply:
aplogies - I meant "AliceBlue" will not work